<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tong_ji</class>
 <widget class="QWidget" name="Tong_ji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tong_ji</string>
  </property>
  <widget class="QListView" name="tong_xian">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="tong_num">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>人数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="tong_ave">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/qian.png);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tong_top">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/qian.png);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tong_bottom">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/qian.png);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tong_tiao">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="maxVisibleItems">
    <number>12</number>
   </property>
   <item>
    <property name="text">
     <string>超过平均学生信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>不及格学生信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>分数段学生信息</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="score_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>162</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>60</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_4">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>162</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>162</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>80</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>162</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>人数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ti_shi_zz">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分数输入不对嗷~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>label_13</zorder>
  <zorder>tong_xian</zorder>
  <zorder>tong_num</zorder>
  <zorder>tong_ave</zorder>
  <zorder>tong_top</zorder>
  <zorder>tong_bottom</zorder>
  <zorder>tong_tiao</zorder>
  <zorder>score_1</zorder>
  <zorder>score_4</zorder>
  <zorder>score_3</zorder>
  <zorder>score_2</zorder>
  <zorder>ti_shi_zz</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tong_tiao</sender>
   <signal>activated(int)</signal>
   <receiver>Tong_ji</receiver>
   <slot>tong_tong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>765</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>score_1</sender>
   <signal>editingFinished()</signal>
   <receiver>Tong_ji</receiver>
   <slot>di()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>score_3</sender>
   <signal>editingFinished()</signal>
   <receiver>Tong_ji</receiver>
   <slot>gao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tong_tong()</slot>
  <slot>di()</slot>
  <slot>gao()</slot>
 </slots>
</ui>
